packages/ai-jsx/src/batteries/use-tools.tsx

```diff
@@ -23,12 +23,10 @@ export interface Tool {
   parameters: FunctionParameters;
 
   /**
-   * A function to invoke the tool.
+   * A function/component that invokes the tool.
```

Comment:
Can we provide guidance on when one would want to provide a function vs. a component?

Reply:
I mean...technically, they're always components, and components just happen to be functions. :)

I added a bit more exposition here but skirted your question because I wasn't sure how to answer it independently of "what are AI.JSX components", and that seemed quite a bit broader than what would be appropriate in this doc comment :P